android/fastlane/actions/get_firebase_info.rb

```diff
@@ -0,0 +1,33 @@
+module Fastlane
+  module Actions
+    class GetFirebaseInfoAction < Action
+
+      # Given an environment
+      # this script returns the Firebase App Id (:app),
+      # and the tester groups path file (:groups_file) associated to it.
+
+      FIREBASE_APP_ID = "FIREBASE_APP".freeze
+      FIREBASE_GROUPS_PATH = "FIREBASE_GROUPS_PATH".freeze
+
+      def self.run(params)
+        {
+          app: Actions::GetEnvironmentInfoAction.run(environment: params[:environment])[FIREBASE_APP_ID],
+          groups_file: Actions::GetEnvironmentInfoAction.run(environment: params[:environment])[FIREBASE_GROUPS_PATH],
```

Comment:
I'd add here a prop to use just `groups` prop taking it from the env file, so in the env file you can put like 
`GROUPS ="Devs, QAs, Tuvieja, Tuprima, Tuhermana"
Is easier and more practical than create a file and put in that file just a little string 
You can use the `groups` prop that is a string for the `firebase_app_distribution` action